docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/charts/index.js

```diff
@@ -48,11 +48,19 @@ const getChartData = async (dataFileName, chartRevision) => {
     );
   }
   const fileURL = `${baseUrl}/${dataFileName}.${dataType || 'json'}?t=${chartRevision || ''}`;
+  const requestURLArray = [fileURL];
+
+  if (miscDataURL) {
+    requestURLArray.push(miscDataURL);
+  }
 
   try {
-    const response = await axios.get(fileURL);
-    const { data } = response;
-    return data;
+    const requests = requestURLArray.map(requestURL => axios.get(requestURL));
+
+    const [chartConfigResponse, geoDataResponse] = await axios.all(requests);
+    const data = chartConfigResponse?.data;
+    const miscData = geoDataResponse?.data;
+    return {data, miscData};
   } catch (error) {
```

Comment:
The naming for the supplemental data gets confusing.  It starts off as a "miscellaneous data URL".  Then it's retrieved as "geo data", then it's back to being stored as "miscellaneous."

Calling it "geo data" is wrong; that just reflects the specific bug you're working on right now. I'm also distrustful of the "miscellaneous" name, though it's definitely more appropriate than "geo data".

Reply:
This I actually intended renaming. Missed out. Will fix.

Reply:
@blairlearn Fixed!